ci/post-collect_logs.yml

```diff
@@ -0,0 +1,166 @@
+---
+# Based on https://raw.githubusercontent.com/openstack-k8s-operators/nova-operator/bc10c4f579f8538899ac7bc5f87bfdb62d7042a4/ci/nova-operator-base/playbooks/collect-logs.yaml
+- hosts: all
+  name: Create zuul-output log dir
+  gather_facts: false
+  tasks:
+    - name: Create log dir
+      ansible.builtin.file:
+        path: "{{ ansible_user_dir }}/zuul-output/logs"
+        state: directory
+        mode: "0755"
+
+- hosts: controller
+  name: Collect logs on the controller
+  gather_facts: false
+  tasks:
+    - name: "Set the sto_dir if it isn't already set"
+      ansible.builtin.set_fact:
+        sto_dir: '{{ ansible_env.HOME }}/{{ zuul.project.src_dir }}'
+      when: sto_dir | default('') | length == 0
+
+    - name: "Get vars common to all jobs"
+      ansible.builtin.include_vars:
+        file: "vars-zuul-common.yml"
+
+    - name: "Get scenario-specific vars"
+      ansible.builtin.include_vars:
+        file: "vars-{{ scenario }}.yml"
+
+    - name: "Create log dir"
+      ansible.builtin.file:
+        path: "{{ logfile_dir }}"
+        state: directory
+        mode: "0755"
+
+    - name: "Log into the cluster"
+      ansible.builtin.import_role:
+        name: rhol_crc
+        tasks_from: add_crc_creds.yml
+
+    - name: "Get builds"
+      ansible.builtin.shell:
+        cmd: |
+          echo "*** [INFO] Showing oc get builds" > {{ logfile_dir }}/post_oc_get_builds.log 2>&1
+          oc get builds -oyaml >> {{ logfile_dir }}/post_oc_get_builds.log 2>&1
+          echo "*** [INFO] Showing oc get builds -oyaml" >> {{ logfile_dir }}/post_oc_get_builds.log 2>&1
+          oc get builds -oyaml >> {{ logfile_dir }}/post_oc_get_builds.log 2>&1
+          cat {{ logfile_dir }}/post_oc_get_builds.log
+      ignore_errors: true
+      changed_when: false
+
+    - name: "Get subscription details"
+      ansible.builtin.shell:
+        cmd: |
+          oc get subscriptions > {{ logfile_dir }}/post_oc_get_subscriptions.log 2>&1
+          oc describe subscription service-telemetry-operator >> {{ logfile_dir }}/post_oc_get_subscriptions.log 2>&1
+          cat {{ logfile_dir}}/post_oc_get_subscriptions.log
+
+    - name: "Get smoketest logs"
+      ansible.builtin.shell:
+        cmd: |
+          echo "*** [INFO] Showing oc get all..." > {{ logfile_dir }}/oc_log.log 2>&1
```

Comment:
It looks like this is basically a copy/paste from https://github.com/infrawatch/service-telemetry-operator/blob/master/tests/smoketest/smoketest.sh#L117 It seems a shame to maintain this meatloaf in two places. 

All of these are going to the same "oc_log.log" file, but it looks like we are already capturing the same output in `{{ logfile_dir }}/smoketest.log` below [1]. Am I missing a reason that we'd need to capture all of this twice?


[1] https://github.com/infrawatch/service-telemetry-operator/pull/479/files#diff-84232940de50ce920ee1af83cd272927075b521e821c56a4fd1933f93dd5eb24R26

Reply:
+1

Reply:
Missed in refactoring, thanks for pointing it out. This re-generation of logs was done before I added the updates to improve logging